hack/jsonformat.go

```diff
@@ -13,8 +13,8 @@ func main() {
 		log.Fatal("Usage: jsonformat.go <filename>")
 	}
 
-	byt, err := ioutil.ReadFile(os.Args[1])
-	if err != nil {
+	byt, readErr := ioutil.ReadFile(os.Args[1])
```

Comment:
This error was being shadowed by another error later, and I chose to rename this one and not the one that was aliasing it.